docs/cloud_build_standards.md

```diff
@@ -0,0 +1,94 @@
+
+Please familiarise yourself with
+
+- Tagging Concepts and what services support / don't support [tagging](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html)
+- [s3 naming restrictions](https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html)
+
+### Tagging considerations
+
+- Tags can be applied anytime
+- Cost allocation reports are only available from the point tags were activated
+- After creating a tag [key] for costs, it must be marked activated and added as a cost allocation tag; otherwise it will not be visible in the Cost Explorer
+
+### General Tagging Standards
+
+Below are a list of tagging standards to consider. Not all will be applicable i.e. based on the above shared account concepts (it's hosted in an account with other services), if it's clustered or not.
+
+All of the below should apply however, to all resources where you can.
+
+
+| Type | Description | Usage | Optional |
+| ---- | ------ | ------ | ------|
+| `TYPE` | Service type | CI | No |
+| `ENV` | Environment | dev, prod, uat, ppd | No |
+| `CLUSTER` | Cluster Name | cluster1, cluster2 | Yes |
+| `NODE` | Node within the cluster | node1 | Yes |
+| `APPSTAGE` | Stage of the application and resources associated | Alpha, Beta, Live | Yes |
+| `ACCESS` | Private or Public | private, public | no |
+| `PROJECT-SERVICE` | name of the service | projectx, projecty | No |
+| `PROJECT-PORTFOLIO` | Name of the portfolio | portfoliox | yes |
+| `SERVICE-TYPE` | Type of service | db, webapp, proxy | No |
+| `AZ` | availability zone number | 1a, 1b, 1c | No |
+| `COST-CODE` | cost code if a shared account | 1234-5678 | yes |
+| `REPO` | Repository related to the creation | repo1 | yes |
+| `TEAM` | Name of the team if a shared account | team1 | yes |
+
+### Additional Tagging Naming Standards
+
+| Type | Description | Usage | Optional |
+| ---- | ------ | ------ | ------|
+| `RDS-TYPE` | database type | Postgres, mysql, oracle | RDSTYPE: Oracle | No |
+
+### Automation and Cloud Rules
+
+- There **MUST NOT** be manual changes to any resources everything should be done via code
+- Console access **MUST** be locked down to readonly to stop manual changes or changes being made that are not tracked via code
+- All code **SHOULD** be triggered via a CI job and not run manually
+- All code **MUST** be tested, linted, and planned
+- Terraform **SHOULD** be used to manage cloud infrastructure
+- Baking images into AMI's using packer **SHOULD** be used over running configuration management tools post infrastructure build phase [see here for sharing AMI's between accounts](https://github.com/chrisns/packer-encrypt-copy)
+- Cloud config is preferred where CoreOS is used to bootstrap the OS
```

Comment:
[Cloud config](https://coreos.com/os/docs/latest/cloud-config.html) is preferred where [CoreOS](https://coreos.com) is used to bootstrap the OS